ui/imports/shared/views/TokenListView.qml

```diff
@@ -29,43 +30,24 @@ Rectangle {
         })
     }
 
-    height: visible ? tokenList.height: 0
-    color: Theme.palette.indirectColor1
-    radius: 8
-
-    StatusListView {
-        id: tokenList
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.top: parent.top
-        width: parent.width
-        height: Math.min(433, tokenList.contentHeight)
-
-        model: SortFilterProxyModel {
-            sourceModel: root.assets
-            filters: [
-                ExpressionFilter {
-                    expression: {
-                        var tokenSymbolByAddress = searchTokenSymbolByAddressFn(d.searchString)
-                        return visibleForNetwork && (
-                            symbol.startsWith(d.searchString.toUpperCase()) || name.toUpperCase().startsWith(d.searchString.toUpperCase()) || (tokenSymbolByAddress!=="" && symbol.startsWith(tokenSymbolByAddress))
-                        )
-                    }
-                }
-            ]
-        }
-        delegate: TokenBalancePerChainDelegate {
-            width: ListView.view.width
-            getNetworkIcon: root.getNetworkIcon
-            onTokenSelected: root.tokenSelected(selectedToken)
-        }
-        headerPositioning: ListView.OverlayHeader
-        header: Rectangle {
-            width: parent.width
-            height: childrenRect.height
+    implicitWidth: contentLayout.implicitWidth
+    implicitHeight: contentLayout.implicitHeight
+
+    ColumnLayout {
+        id: contentLayout
+
+        anchors.fill: parent
```

Comment:
This will create a binding loop

Reply:
Not sure I understand why.

`anchors.fill: parent` is here to follow `root`'s size (user-defined size), and I didn't set a size dependency the other way (`root`'s `width`/`height` is not set).

I only set the size hints (in case the user of the `TokenListView` doesn't provide one, the recommended size will be followed. This fits nicely with the dialogue and other containers that use `implicitWidth`/`implicitHeight` hints as the default size.